internal/cli/atlas_whitelist_delete.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"fmt"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/mongodb/mcli/internal/flags"
+	"github.com/mongodb/mcli/internal/prompts"
+	"github.com/mongodb/mcli/internal/store"
+	"github.com/mongodb/mcli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type atlasWhitelistDeleteOpts struct {
+	*globalOpts
+	entry   string
+	confirm bool
+	store   store.ProjectIPWhitelistDeleter
+}
+
+func (opts *atlasWhitelistDeleteOpts) init() error {
+	if opts.ProjectID() == "" {
+		return errMissingProjectID
+	}
+
+	s, err := store.New()
+
+	if err != nil {
+		return err
+	}
+
+	opts.store = s
+	return nil
+}
+
+func (opts *atlasWhitelistDeleteOpts) Run() error {
+	err := opts.store.DeleteProjectIPWhitelist(opts.ProjectID(), opts.entry)
+
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("Project whitelist entry '%s' deleted\n", opts.entry)
+
+	return nil
+}
+
+func (opts *atlasWhitelistDeleteOpts) Confirm() error {
+	if opts.confirm {
+		return nil
+	}
+	prompt := prompts.NewDeleteConfirmation(opts.entry)
+	return survey.AskOne(prompt, &opts.confirm)
+}
+
+// mcli atlas whitelist delete <entry> --force
+func AtlasWhitelistDeleteBuilder() *cobra.Command {
+	opts := &atlasWhitelistDeleteOpts{
+		globalOpts: newGlobalOpts(),
+	}
+	cmd := &cobra.Command{
+		Use:     "delete [entry]",
```

Comment:
[q] Does the --force show up in the help text if we don't put it here?

Reply:
yes cause that's a flag and all flag related help/autocomplete info comes from the `cmd.Flags()` property, if you don't want it to show then you need to use `MarkHidden`